### DISCUSSION TITLE
I hope we can use extenal editors.

### DISCUSSION THREAD

**Discussion by otakutyrant** [violentmonkey/violentmonkey#1832] [Ideas]
I want to use NeoVim! I think the feature may be simple: open a script file with external editor (like my favorite neovim in Linux environment), and keep the former locked untile the external editor windows is closed. Then users can reload the new script!

**Comment by tophf** [comment]
It's already possible: https://violentmonkey.github.io/posts/how-to-edit-scripts-with-your-favorite-editor/
